### DISCUSSION TITLE
Payment Request QR Code

### DISCUSSION THREAD

**Discussion by SpeedyCoinzales** [btcpayserver/btcpayserver#3492] [Ideas / Feature Requests]
Hi,
At the view site of a payment request, there is only the copy link. My customer has a need to print the link on their paper invoices. The link isn't suitable for this use case. Thanks

**Comment by pavlenex** [comment]
Thanks for reporting. Recently we've talked about simplifying that particular page, and making it similar to invoices page in terms of information architecture and UI. 

For the reference this is the page in question cc @dstrukt 
![Screenshot 2022-02-25 at 10 59 26](https://user-images.githubusercontent.com/36959754/155677829-60d6fc9a-f832-4aad-aab3-2bb2d64c1c5b.png)

**Reply by bolatovumar** [reply]
@pavlenex I think @SpeedyCoinzales meant the payment request page itself rather than the page listing all payment requests. Is this correct @SpeedyCoinzales?

**Reply by SpeedyCoinzales** [reply]
Hi @bolatovumar @pavlenex, Yes @bolatovumar is right. Please, have a look on the following screenshot. Something similar to this...
<img width="1169" alt="Payment request" src="https://user-images.githubusercontent.com/99530952/155878534-f812c504-8b9b-4c74-9688-565699effba7.png">

**Reply by bolatovumar** [reply]
@SpeedyCoinzales feels like for this use case it would make sense to have the QR code present only in the print version of the page?

**Reply by SpeedyCoinzales** [reply]
Hi, as the most firms has their own invoice system and forms + fiat currency to be accepted, less will use the print option of BTCPAY. They want to copy and paste the jpeg or png into their invoice. If you want them to use this option, you must build up an easy to use and customized invoice form with fiat bank account, logos etc.. This make no sense at the moment, in my opinion. Best regards
